src/store-api/src/metadata.rs

```diff
@@ -741,92 +718,114 @@ impl RegionMetadataBuilder {
         Ok(())
     }
 
-    fn change_column_inverted_index_options(
-        &mut self,
-        column_name: String,
-        value: bool,
-    ) -> Result<()> {
+    fn set_indexes(&mut self, options: Vec<SetIndexOption>) -> Result<()> {
+        let mut index_options = options
+            .into_iter()
+            .map(|option| (option.column_name().clone(), option))
+            .collect::<HashMap<_, _>>();
```

Comment:
What if there are multiple requests to one column? looks like only one will take effect?

Reply:
Good catch, resolved in [6e9b769](https://github.com/GreptimeTeam/greptimedb/pull/6487/commits/6e9b76952503f612e578e264168d493a9143ef09)